resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,836 @@
+
+To advertise additional prefixes to GCP, < insert instructions here>.
+
+```
+<insert configuration code snippet here>
+```
+
+### Saving the configuration
+
+Follow the procedure in this section to save the on-premises configuration.
+
+< insert the instructions for saving the configuration here.>
+
+```
+<insert configuration code snippet here>
+```
+
+### Testing the configuration
+
+It's important to test the VPN connection from both sides of a VPN tunnel. For either side, make sure that the subnet that a machine or virtual machine is located in is being forwarded through the VPN tunnel.
+
+First, create VMs on both sides of the tunnel. Make sure that you configure the
+VMs on a subnet that will pass traffic through the VPN tunnel.
+
+-  Instructions for creating virtual machines in Compute Engine are located
+in the 
+[Getting Started Guide](https://cloud.google.com/compute/docs/quickstart).
+-  Instructions for creating virtual machine for <vendor-name><product-name>
+platforms are located at <link here>.
+
+After VMs have been deployed on both the GCP and <vendor-name><product-name>
+platforms, you can use an ICMP echo (ping) test to test network connectivity
+through the VPN tunnel.
+
+On the GCP side, use the following instructions to test the connection to a
+machine that's behind the on-premises gateway:
+
+1. In the GCP Console,
+[go to the VM Instances page](https://console.cloud.google.com/compute?).
+1. Find the GCP virtual machine you created.
+1. In the **Connect** column, click **SSH**. A browser window opens at the VM
+command line.
+1. Ping a machine that's behind the on-premises gateway.
+
+<Insert any additional instructions for testing the VPN tunnels from the <vendor
+name><product-name> here. For example, below is an example of a successful ping
+from a Cisco ASR router to GCP.>
+
+    cisco-asr#ping 172.16.100.2 source 10.0.200.1
+    Type escape sequence to abort.
+    Sending 5, 100-byte ICMP Echos to 172.16.100.2, timeout is 2 seconds:
+    Packet sent with a source address of 10.0.200.1
+    !!!!!
+    Success rate is 100 percent (5/5), round-trip min/avg/max = 18/19/20 ms
+
+
+## Troubleshooting IPsec on \<vendor-name>\<product-name>
+
+For troubleshooting information, see the <vendor-name><product-name>
+troubleshooting guide <add link>.
+
+<Add details here about what kind of troubleshooting information can be found in
+the <vendor-name><product-name> guide.>
+
+
+## Reference documentation
+
+You can refer to the following <vendor-name><product-name> documentation and
+Cloud VPN documentation for additional information about both products.
+
+### GCP documentation
+
+To learn more about GCP networking, see the following documents:
+
+-  [VPC Networks](https://cloud.google.com/vpc/docs)
+-  [Cloud VPN Overview](https://cloud.google.com/compute/docs/vpn/overview)
+-  [Creating Route-based VPNs](https://cloud.google.com/vpn/docs/how-to/creating-route-based-vpns)
```

Comment:
Since this is HA VPN, I'd remove the Classic VPN Route-based and Policy-based references and add the Cloud VPN topologies page: https://cloud.google.com/vpn/docs/concepts/topologies.

I'd also add the page to check VPN status, which includes checking for HA:
https://cloud.google.com/vpn/docs/how-to/checking-vpn-status